llms/mlx_lm/utils.py

```diff
@@ -404,6 +404,30 @@ def load_config(model_path: Path) -> dict:
     return config
 
 
+def compute_bits_per_weight(config, weights):
+    weight_count = 0
+    bit_count = 0
+    for name, param in weights.items():
+        bit_count += param.size * param.dtype.size * 8
```

Comment:
Nit:
```suggestion
        bit_count += param.nbytes * 8
```

Reply:
Nice idea! It came out much cleaner this way.
Maybe I'm misunderstanding but can we check for `to_quantized` here? That signifies that the module is _not_ quantized, but not all unquantized layers implement that method.

Reply:
Nice, it looks great. And yea you're right.. that was a bad suggestion.